components/collaboration/UserResponse.tsx

```
@@ -0,0 +1,183 @@
+                <FontAwesomeIcon icon={faPaperclip} />
+                <Text
+                  key={index}
+                  style={{
+                    fontSize: 16,
+                  }}
+                >
+                  {attachment.name}
+                </Text>
+              </View>
+            );
+          })}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)